src/components/Booking/Booking.jsx

```diff
@@ -0,0 +1,24 @@
+const Booking = (props) => {
+  const {
+    title,
+    firstName,
+    surname,
+    email,
+    roomId,
+    checkInDate,
+    checkOutDate,
+  } = props.booking;
+  return (
+    <tr data-testid="booking-component">
+      <td>
+        {title} {firstName} {surname}
+      </td>
+      <td>{email}</td>
+      <td>{roomId}</td>
+      <td>{checkInDate}</td>
+      <td>{checkOutDate}</td>
+    </tr>
+  );
+};
+
+export default Booking;
```

Comment:
this code looks good. It is easy to understand and neat and clean. 
In my opinion it would look better if you had less lines of code, it would look shorter then. So the const you could write in one line like const {title, firstName.....} . 
But if it easier for you to work like this then that is fine.